src/drake/shell.clj

```diff
@@ -96,7 +102,7 @@
         proc (.exec (Runtime/getRuntime)
```

Comment:
I've tried:

- shell scripts that invoke a local script 
- ruby scripts that require a local script
- trying to test a clojure require, but can't get it working with c4rows protocol. Does that protocol even support requires?